src/main.cpp

```diff
@@ -76,48 +76,25 @@ void deinit()
 void setupPHYSFS()
 {
 	FileSystem& fs = FileSystem::getSingleton();
-	
-	// Game should be playable out of the source package on all
-	// relevant platforms.
-	#if BLOBBY_ON_DESKTOP
-		std::string baseSearchPath("data" + fs.getDirSeparator());
-	#else
-		std::string baseSearchPath(fs.getBaseDir());
-	#endif
 
-	/*
-	set write dir
-	*/
+	std::string baseSearchPath = fs.getDataDir();
+
+	// setup write directory
 	std::string writeDir = fs.getPrefDir();
 	fs.setWriteDir(writeDir);
 	fs.probeDir("replays");
-	fs.probeDir("gfx");
-	fs.probeDir("sounds");
-	fs.probeDir("scripts");
-	fs.probeDir("backgrounds");
-	fs.probeDir("rules");
-
-	/*
-	set read dir (local files next to application)
-	*/
+
+	// and add all reading locations
 	fs.addToSearchPath(baseSearchPath);
```

Comment:
I'm not sure if I like this. We split a repeating path in two separate files (src/FileSystem.cpp - appendToSearchDirs)